keras/engine/saving.py

```diff
@@ -558,16 +558,25 @@ def load_attributes_from_hdf5_group(group, name):
     return data
 
 
-def save_weights_to_hdf5_group(f, layers):
+def save_weights_to_hdf5_group(group, layers):
+    """Saves weights into the HDF5 group.
+
+    # Arguments
+        group: A pointer to a HDF5 group.
+        layers: Layers to load.
+
+    # Returns
```

Comment:
Keras functions which don't return anything don't have a `# Returns` in their docstring.